src/mouseButtonNumbers.ts

```diff
@@ -1,8 +1,7 @@
 export const mouseButtonNumbers = {
-    "none": 0,
```

Comment:
Since my plan is to track the state - there should be no way to configure "none" mouse buttons for commands like `realMouseDown` and stuff. Unless this is needed by some kind of mouse input virtualization but I would expect such things to be pretend that some button is pressed so I'm not sure when in practice the none button could be actually used